src/image/mod.rs

```diff
@@ -370,3 +376,16 @@ impl<'de> Deserialize<'de> for Arch {
         Ok(arch.as_str().into())
     }
 }
+
+impl Default for Arch {
+    fn default() -> Self {
+        // Translate from the Rust architecture names to the Go versions.
+        // I think the Rust ones are the same as the Linux kernel ones.
+        let goarch = match std::env::consts::ARCH {
+            "x86_64" => "amd64",
+            "aarch64" => "arm64",
```

Comment:
Can we do this for all of them listed here: https://doc.rust-lang.org/std/env/consts/constant.ARCH.html

Reply:
This mapping contains required transformations.  Can you post a quick diff/patch for what you're suggesting?  Do you want to add an explicit e.g. `s390x` => `s390x` (i.e. for cases that aren't transformed?)

Reply:
Ah oh, I missed the line below. All fine!